test/services/worker_task_tracking_repository.js

```diff
@@ -68,4 +81,11 @@ describe('Worker Log Repository', () => {
     await expect(storage.finishWork(fakeTaskId)).to.be.fulfilled;
     await expect(storage.tryToBeginWork(exampleTaskType)).to.be.rejectedWith('Work of this type is currently in progress');
   });
+
+  it('creates index with correct ttl', async () => {
```

Comment:
Repository does not create index, migration do. Not sure if this test should be here.

Reply:
++ @Anze1m. The name of the tests is at best misleading. Also: the only reason this tests pass is because someone run a migration once (cleanDatabase doesn't remove indexes).

So IMO: If the repository assumes that some index is necessary for it to work properly, it should check for its presence.